api/users/constants.py

```diff
@@ -1 +1,9 @@
 DEFAULT_DELETE_ORPHAN_ORGANISATIONS_VALUE = False
+
+ALLOWED_UTM_KEYS = {
+    "utm_source",
```

Comment:
My gut feeling with this was whether to just blindly send up utm_x params, assuming the BE just ignores them.